Libraries/LibCore/CSocket.cpp

```diff
@@ -31,6 +33,15 @@ bool CSocket::connect(const String& hostname, int port)
     return connect(host_address, port);
 }
 
+void CSocket::set_blocking(bool blocking)
+{
+    int flags = fcntl(fd(), F_GETFL, 0);
```

Comment:
Let's make a habit to error check syscalls.
We never know when the kernel might do something erratic, so it's best to fail early.
Even just asserting that nothing went wrong is better than nothing :)